tests/extra_source_and_tests/extra_src/shared/pressure_boundary/kernel_summation.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "kernel_summation.h"
+
+namespace SPH
+{
+//=================================================================================================//
+template <class DataDelegationType>
+template <class BaseRelationType>
+NablaWV<DataDelegationType>::NablaWV(BaseRelationType &base_relation)
+    : LocalDynamics(base_relation.getSPHBody()), DataDelegationType(base_relation),
+      kernel_sum_(*this->particles_->template registerSharedVariable<Vecd>("KernelSummation"))
+ {}
+//=================================================================================================//
+} // namespace SPH
```

Comment:
This file seems also not essentially needed.

Reply:
temporarily reserve it, this class (.h .cpp. hpp files) will be rearranged later